core/state/stateupdate.go

```diff
@@ -33,6 +35,7 @@ type contractCode struct {
 type accountDelete struct {
 	address        common.Address         // address is the unique account identifier
 	origin         []byte                 // origin is the original value of account data in slim-RLP encoding.
+	storages       map[common.Hash][]byte // storages stores mutated slots, the value should be nil.
```

Comment:
What do you mean by "should be nil"? Do you mean that it's a programming error otherwise, or that something abnormal, like destruct+resurrect occurred?